frontend/docs/admin-guide/getting-started.md

```diff
@@ -4,66 +4,71 @@ outline: deep
 
 # High Level Overview
 
-What is {{ $params.APPLICATION_NAME }}?
+## What is {{ $params.APPLICATION_NAME }}?
```

Comment:
Did you verify this works with replacing variables? I had an issue before where `{{ $params.APPLICATION_NAME }}` could not be added as a header because it would mess up our search functionality in the docs. You can verify this by trying to use the search bar in the top nav bar